content/blog/2022-05-05-e2e-cv-pt3.md

```diff
@@ -0,0 +1,289 @@
+---
+title:
+  'End-to-End Computer Vision API, Part 3: Remote Experiments & CI/CD For
+  Machine Learning'
+date: 2022-05-05
+description: >
+  This is the third and last part of this series of posts on how to build an
+  end-to-end Computer Vision solution according to the best available MLOps
+  practices. 
+
+  In part 1, we talked about exploratory work in Jupyter Notebooks; versioning
+  data in remote storage with DVC; and refactoring the code from Jupyter
+  Notebooks into DVC pipeline stages.
+
+  Part 2 talked about the process of managing experiments with DVC pipelines,
+  DVCLive and Iterative Studio.
+
+descriptionLong: |
+
+  This is the third and last part of this series of posts on how to build an end-to-end Computer Vision solution according to the best available MLOps practices. 
+
+  In part 1, we talked about exploratory work in Jupyter Notebooks; versioning data in remote storage with DVC; and refactoring the code from Jupyter Notebooks into DVC pipeline stages.
+
+  Part 2 talked about the process of managing experiments with DVC pipelines, DVCLive and Iterative Studio.
+
+picture: <TBD>
+author: alex_kim
+commentsUrl: https://discuss.dvc.org/t/end-to-end-computer-vision/1178
+tags:
+  - Computer Vision
+  - DVC
+  - CML
+  - Studio
+  - CI/CD
+  - Experiment Tracking
+---
+
+### Introduction
+
+In this post, we’ll focus on leveraging cloud infrastructure with CML and how to
+enable automatic reporting (with metrics, plots, and other visuals) to accompany
+every commit/pull request on GitHub.
+
+### Leveraging Cloud Resources with CI/CD and CML
+
+If you ever wondered if there's an easy way to quickly and easily:
```

Comment:
It might be better to go back to addressing a problem from the use case. For example, since CV models often need powerful GPUs, we want to train in the cloud.